apps/nativeapp/app/screens/Home/Home.tsx

```
@@ -247,82 +250,82 @@ const Home = ({navigation, route}) => {
-              }
+              ...oldData,
+              json: {
+                ...oldData?.json,
+                data: oldData?.json?.data?.map(item =>
+                  item.id === res?.json?.data?.id ? res?.json?.data : item,
+                ),
+              },
+            }
             : null,
       );
       if (req?.json?.body.hasOwnProperty('isMonitored')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)